lib/nerves_hub/deployments.ex

```diff
@@ -154,19 +164,23 @@ defmodule NervesHub.Deployments do
   def broadcast(deployment, event, payload \\ %{})
 
   def broadcast(:none, event, payload) do
-    Phoenix.PubSub.broadcast(
-      NervesHub.PubSub,
-      "deployment:none",
-      %Phoenix.Socket.Broadcast{event: event, payload: payload}
-    )
+    payload = %Phoenix.Socket.Broadcast{
+      topic: "deployment:none",
+      event: event,
+      payload: payload
+    }
+
+    Phoenix.PubSub.broadcast(NervesHub.PubSub, "deployment:none", payload)
```

Comment:
```suggestion
    msg = %Phoenix.Socket.Broadcast{
      topic: "deployment:none",
      event: event,
      payload: payload
    }

    Phoenix.PubSub.broadcast(NervesHub.PubSub, "deployment:none", msg)
```

Just a nit to not overload the payload meaning within the same function context